content/introduction.html

```diff
@@ -22,23 +22,36 @@ <h1> Preface </h1>
 
 <p class="full">
 The findings in this report were derived from the results of a class survey.
-In total, 113 out of 141 (80%) graduating students responded. SE classes
-can vary significantly depending on the year, so generalize from this survey
-to other classes at your own risk.
+In total, 66 out of 92 (72%) graduating students, and 7 who were in SE but
+transferred out, responded. An addendum survey was also sent out, where
+45 more responses were retrieved from a subset of the previous respondants.
```

Comment:
do we know necessarily that the 45 respondents were a subset 😮 ?

Reply:
No :( lol